pom.xml

```diff
@@ -672,6 +672,13 @@
 			<artifactId>gson</artifactId>
 			<version>${com.google.code.gson.version}</version>
 		</dependency>
+		<!-- https://mvnrepository.com/artifact/com.monitorjbl/xlsx-streamer -->
+		<dependency>
+			<groupId>com.monitorjbl</groupId>
+			<artifactId>xlsx-streamer</artifactId>
+			<version>${com.monitorjbl.xml.streamer.version}</version>
```

Comment:
What is the advantages of this over using apache poi which we already use?

Reply:
Apache POI crashes when creating a workbook if the column data doesn't match the column type. For example the amr detection pipeline outputs an excel file with column types set to `Number` but the data is `Text`. After discussing this with Tom I found this package to help us :)

Reply:
Perfect, just wanted to know, Thanks.